Components/Memory/CMakeLists.txt

```diff
@@ -6,23 +6,28 @@ message(DEBUG "QUICC_CURRENT_COMPONENT_LIB_NAMESPACE: ${QUICC_CURRENT_COMPONENT_
 
 add_library(${QUICC_CURRENT_COMPONENT_LIB} STATIC
   Memory/Cpu/NewDelete.cpp
+  Memory/Cpu/Pool.cpp
   Memory/Cuda/Malloc.cu
+  Memory/Cuda/Pool.cu
 )
 
+set_property(TARGET ${QUICC_CURRENT_COMPONENT_LIB}
+  PROPERTY POSITION_INDEPENDENT_CODE ON)
+
 target_include_directories(${QUICC_CURRENT_COMPONENT_LIB}
   PUBLIC
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>"
     "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
 )
 
-
-set(PHS
-  Memory/Memory.hpp
-  Memory/MemoryResource.hpp
-  Memory/Pensieve.hpp
-  Memory/Cpu/NewDelete.hpp
-  Memory/Cuda/Malloc.hpp
-)
+# set(PHS
```

Comment:
No headers?

Reply:
they are installed with the directory, there are are ways but I got stuck and I wanted to move on!
I'll remove the comments